Jenkinsfile

```diff
@@ -33,25 +33,14 @@ node('heavy && linux && docker') {
         sh 'npm run build'
       }
 
-      stage('Generate docs') {
-        sh 'npm run doc:generate'
-      }
-    }
-
-    stage('Clean working directory') {
-      sh 'git checkout -- .'
-      sh 'git clean -f'
-    }
```

Comment:
This was originally added because `npm run build` would occasionally update the README (see https://github.com/coveo/ui-kit/issues/189#issuecomment-733037682). It shouldn't be necessary anymore.

If it turns out that this is necessary, then a cleanup step should be added to `npm:publish` IMO.